cybersyn/locale/en/base.cfg

```diff
@@ -23,7 +23,7 @@ cybersyn-locked-slots=The default number of locked slots per cargo wagon when a
 cybersyn-network-flag=The default set of sub-networks a station will service when no network signal is given to a station. This integer is interpretted bit-wise to give 32 possible sub-networks to choose from.
 cybersyn-fuel-threshold=What percentage of a train's fuel inventory must be full to skip refueling. If this is set to 1, trains will always visit a refueler after completing a delivery.
 cybersyn-depot-bypass-enabled=If checked, when a train completes a delivery and refueling, it may take a new order from the network before having to return to its depot.
-cybersyn-warmup-time=How many seconds a cybernetic combinator will wait before connecting to the Cybersyn network. This is a grace period to modify or correct the circuit network before trains start dispatching to a newly blueprinted station.
+cybersyn-warmup-time=How many seconds a cybernetic combinator will wait before connecting to central planner. This is a grace period to modify or correct the circuit network before trains start dispatching to a newly blueprinted station.
```

Comment:
@mamoniot What do you think about this?

Reply:
I'm going to revert this, because I miss the main idea of cybersyn (reference to IRL project).